### PR TITLE
Add prettier formater

### LINKED ISSUE #11: Add Prettier config file
To maintain consistent coding styles between all developers, we can use Prettier.

I think these are the steps:
1. Install prettier to the project: https://prettier.io/docs/en/install.html
1. Add a `.prettierrc` config file to the root of the project with the following JSON:
   ```
   {
     "arrowParens": "avoid",
     "jsxSingleQuote": true,
     "singleQuote": true,
     "trailingComma": "all"
   }
   ```
1. Install the VS Code Prettier extension: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode
1. Confirm it works by making some changes to a file and saving—the file should be re-formatted by Prettier

### PR DESCRIPTION
**Pull request by viktoriia-kosenko**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "arrowParens": "avoid",
+  "jsxSingleQuote": true,
```
Comment (bonboh) [inline .prettierrc]: I just tried this out in VS Code and we should delete this line! `"jsxSingleQuote": true,`

**Review by bonboh — CHANGES_REQUESTED** [review]
When testing this out, I realised we also need to add another file. Can you add the following JSON to this new file `.vscode/settings.json`?

```json
{
  "editor.formatOnSave": true
}
```

File: .prettierrc (hunk 1 above)
Comment (viktoriia-kosenko) [reply .prettierrc]: Done!

**Review by bonboh — APPROVED** [review]
👍

**Pull request merged**